tasks.ps1

```diff
@@ -46,11 +46,11 @@ else {
 if ($deployYamlsFound.Count -gt 0) {
   New-Item -ItemType Directory -Force -Path "/tmp/$env:GITHUB_REPOSITORY"
   Set-Location "/tmp/$env:GITHUB_REPOSITORY"
-
+  $CakeLogLevel = if ($env:CAKE_LOG_LEVEL) { $env:CAKE_LOG_LEVEL } else { "Information" };
   dotnet nuget add source --name cake --username "${NugetUser}" --password "${NugetToken}" --store-password-in-clear-text "https://nuget.pkg.github.com/variant-inc/index.json"
   ce dotnet nuget update source cake -u "${NugetUser}" -p "${NugetToken}" --store-password-in-clear-text -s "https://nuget.pkg.github.com/variant-inc/index.json"
   ce dotnet new tool-manifest --force
   ce dotnet tool install --version "${TaskRunnerVersion}" --no-cache Variant.Cake.Runner
-  ce dotnet variant-cake-runner --target CreateRelease --path $variantApiDeployYamlPath
+  ce dotnet variant-cake-runner --target CreateRelease --path $variantApiDeployYamlPath --logLevel $CakeLogLevel
```

Comment:
I also added Verbosity flag to Cake if you want to add that

Reply:
Yeah this release includes the Verbosity flag in the cake args in cake-runner